src/main/resources/static/js/createSurvey.js

```diff
@@ -23,7 +24,37 @@ rangeInputForm =`
 	 <input type="number" id="rangeQuestionMax" name="questionMax"> <br>
      <button type="button" id="addRangeQuestion">Add Question</button>
     </div>`
+	
+	
+//Input HTML for creating a MC based question
+MCInputForm =`
+    <br>
+    <div id="questionInputBlock">
+     <label for="MCQuestionPrompt">Question:</label><br>
+     <textarea id="MCQuestionPrompt" name="questionPrompt" rows="2" cols="80"></textarea><br>
+	 <label for="MCChoicePrompt">Choice:</label><br>
+     <textarea id="MCChoice" name="MCChoice" rows="2" cols="80"></textarea><br>
+	 <button type="button" id="addChoice">Add Choice</button>
+	 <button type="button" id="deleteChoices">Delete All Choices</button>
+	 <button type="button" id="addMCQuestion">Add Question</button>
+	 
+    </div>`
 
+// Input HTML for setting the choices of a MC question
```

Comment:
Can the commented out stuff be removed or is it supposed to be uncommentted?

Reply:
Sorry, I just forgot to delete it